proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,535 @@
+   wurden.
+2. Sie wählt den Vorstand aus dem Kreis der Kernmitgliedern und kann
+   Vorstandsmitglieder abwählen.
+3. Sie entscheidet über Einsprüche gegen den Ausschluss von Mitgliedern.
+4. Sie nimmt den Jahresbericht des Vorstandes und den Bericht der
+   Rechnungsprüfer entgegen und erteilt dem Vorstand Entlastung.
+5. Sie beschließt über Satzungsänderungen und die Auflösung des Vereins.
+6. Ihr sind insbesondere die Jahresrechnung und der Jahresbericht zur
+   Beschlussfassung über die Genehmigung und die Entlastung des Vorstandes
+   schriftlich vorzulegen. Sie beruft einen Rechnungsprüfer, die weder dem
+   Vorstand noch einem vom Vorstand berufenen Gremium angehören und auch nicht
+   beim Verein angestellt ist, um die Buchführung einschließlich Jahresabschluss
+   zu prüfen und über das Ergebnis vor der Mitgliederversammlung zu berichten.
+   Dr Rechnungsprüfer hat Zugang zu allen Buchungs- und Rechnungsunterlagen des
+   Vereins.
+7. Sie beschließt über Käufe, Verkäufe, Investitionen und die Aufnahme von
+   Krediten.
+
+## 8. Der Vorstand
+
+1. Der Vorstand besteht aus dem Vorsitzenden, dem zweiten Vorsitzenden und dem
+   Schatzmeister die für eine Amtszeit von 3 Jahren gewählt werden.
+
+   TODO: Definieren Sie die Verantwortlichkeiten der Rollen.
+
+2. Die Aufgaben des Vorstands und der Ablauf der Sitzungen sind wird durch eine
+   Geschäftsordnung des Vorstands geregelt.
+3. Der Vorstand beschließt über alle Vereinsangelegenheiten, soweit sie nicht
+   eines Beschlusses der Mitgliederversammlung bedürfen. Er führt die Beschlüsse
+   der Mitgliederversammlung aus.
+4. Jedes Vorstandsmitglied hat das Recht, den Verein zu vertreten.
+5. Im Falle des Ausscheidens eines Vorstandsmitglieds während der Amtszeit
+   bestimmt der Vorstand auf Basis einer Internetwahl ein kommissarisches
+   Vorstandsmitglied. Das kommissarische Vorstandsmitglied bleibt bis zur
+   nächsten Mitgliederversammlung im Amt. Über eine endgültige Nachfolge im
+   Vorstand entscheidet die Mitgliederversammlung.
+6. Im Falle des Ausschlusses eines Vorstandsmitglieds gemäß §4 (5) wird dieses
+   durch ein kommissarisches Vorstandsmitglied ersetzt.
+7. Der Vorstand kann zur Führung der laufenden Geschäfte einen hauptamtlichen
+   Geschäftsführer bestellen.
+8. Der Geschäftsführer nimmt an den Mitgliederversammlungen und
+   Vorstandssitzungen teil.
+9. Der Vorstand kann die Satzung aus formalen Gründen ändern, wenn von den
+   Behörden verlangt wird.
+
+## 9. Geschäftsordnung
+
+Die Geschäftsordnung regelt über den Rahmen der Satzung hinausgehende Details
+der Vereinsarbeit. Der Vorstand setzt die Geschäftsordnung nach der Zustimmung
+der Kernmitglieder in einer Internetabstimmung in Kraft.
+
+## 10. Protokolle
+
+Alle Beschlüsse des Vorstandes und der Mitgliederversammlung werden schriftlich
+in deutscher Sprache dokumentiert und stehen den Mitgliedern zur Einsichtnahme
+und zur Prüfung der Beschlüsse durch das Registergericht zur Verfügung.
```

Comment:
Is it necessary to document daily board decisions in German language too?

Reply:
No. only tax relevant things. I think it is more easy to have everything in German. 
Please also not that our daily Mixxx work is to a Beschluss, we need a Beschluss when we spend money and than it needs to be in German anyway.

Reply:
I don't think, that the board of an e.V. need to do written Beschluss by law, if something has to be paid. This might be required by the Geschäftsordnung of the e.V., but the tax office requires only that the purpose of each payment is documented.

Reply:
Ok, I don't know the implications. At least from the KDE e.V. I have learned that they keep their protocols in German. Do you have an alternative sentences that works.

Reply:
What is the proposal? My idea was that we discuss everything in English but keep the protocols in German for legal reasons. Everyone can paste them to DeepL and has a translation in their Mother language fast without extra effort put on the Protokollführer.

Reply:
Not sure if the on-the-fly translation is too much for the Protokollführer. I think it should be up to them to do however they're most proficient and if they don't actually speak german we can help them translate afterwards...
Anyways, it'll probably have to be something like you proposed where we have german protocols for legal reasons and then leverage auto translation for everything else.

Reply:
All documents for the court and tax office must be in German language by law. The authorities would only accept translations by certified translators, which are very expensive. Therefore the original of all these documents should be in German language. Than we can create informal translation to English ourself, or using an online translator.

Mandatory documents for the authority are at least:
-Gründungsdokumente
-Die Satzung
-Änderungen an der Satzung
-Protokoll der Mitgliederversammlung (incl. Vorstandswahlen, Entlastung des Vorstands, Haushaltsplan, Anträge und Beschlüsse)
-Jahresabschluss mit Bilanz, und Gewinn- und Verlustrechnung
-Tätigkeitsbericht (anhand der tatsächlichen Tätigkeiten des Vereins wird über das Fortbestehen der Gemeinnützigkeit entschieden)
-Vermögensaufstellung
-Mitteilungen über die Anschrift des Wohnsitzes der Vorstandsmitglieder an das Registergericht bei Änderungen

But the daily decisions of the board do not need to be in written form by law. But it make sense to do this for transparency to the own members.

Reply:
I didn't know that. Thanks for the clarification. I see our hands are pretty much tied then.